src/main/groovy/com/cloudcard/photoDownloader/SqsPhotoService.groovy

```diff
@@ -0,0 +1,136 @@
+package com.cloudcard.photoDownloader
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sqs.SqsClient
+import software.amazon.awssdk.services.sqs.model.DeleteMessageRequest
+import software.amazon.awssdk.services.sqs.model.Message
+import software.amazon.awssdk.services.sqs.model.ReceiveMessageRequest
+import software.amazon.awssdk.services.sqs.model.SqsException
+
+import javax.annotation.PostConstruct
+
+import static com.cloudcard.photoDownloader.ApplicationPropertiesValidator.throwIfBlank
+
+@Service
+@ConditionalOnProperty(value = "downloader.photoService", havingValue = "SqsPhotoService", matchIfMissing = true)
+class SqsPhotoService implements PhotoService {
+
+    private static final Logger log = LoggerFactory.getLogger(SqsPhotoService.class)
+
+    @Value('${sqsPhotoService.queueUrl}')
+    String queueUrl
+
+    @Value('${sqsPhotoService.pollingIntervalSeconds}')
```

Comment:
@tonyerskine we think adding in a default here would make things easier for an implementer. @slackboxster agrees.